xbmc/addons/kodi-addon-dev-kit/src/api2/kodi-addon-sharedlibrary/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+project(kodi-addon-sharedlibrary-api2)
+
+cmake_minimum_required(VERSION 3.0.0)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR})
+
+enable_language(C CXX)
```

Comment:
can be removed, that's the default. if needed language can be specified as part of project()